src/shared/containers/StudioContainer.tsx

```diff
@@ -46,11 +51,46 @@ const StudioContainer: React.FunctionComponent<StudioContainerProps> = ({
         // TODO: show a meaningful error to the user.
       });
   }, [orgLabel, projectLabel, studioId]);
+
+  const updateStudio = async (label: string, description?: string) => {
+    if (studioResource) {
+      await nexus.Resource.update(
+        orgLabel,
+        projectLabel,
+        studioId,
+        studioResource._rev,
+        {
+          label,
+          description,
+        },
+      ).then(response => {
+        notification.success({
+          message: 'Studio was edited successfully',
+          duration: 2,
+        });
+      }).catch(error => {
+        notification.error({
+          message: 'An error occurred',
+          description: error.reason || error.message,
+          duration: 3,
+        });
+      });
+    }
+  }
+
   return (
     <>
       {studioResource ? (
         <>
-          <h1 className="title">{studioResource.label}</h1>
+          <div className="title-container">
```

Comment:
This would be cool as a presentational component, then you can remove the styles from `App.less` and put it somewhere more specific